tox.ini

```diff
@@ -33,8 +33,10 @@ deps = flake8
 commands = flake8
 
 [testenv:readme]
-deps = readme_renderer
-commands = python setup.py check --restructuredtext --strict
+deps = twine
+commands =
+    {envpython} setup.py -q sdist bdist_wheel
+    twine check dist/*
```

Comment:
I took this over from PR #148. This is the recommended replacement for the deprecated `check` call on setuptools.